### ISSUE TITLE
Files not visible but properly scanned

### ISSUE THREAD

**Issue by Denniz** [airsonic/airsonic#743]
## Problem description

An entire folder with files is no longer visible. Airsonic scans the folder properly and shows the correct amount of songs (just over 3k) in the left side of the interface. But clicking "Home" at the top and "all" in the home screen doesn't show anything.

### Steps to reproduce

After noticing this I've renamed the `/var/airsonic/` folder and restarted the application to ensure a clean setup. Then I've added only this 1 folder that is causing problems and again it scans the 3k+ files and shows the correct numbers, but the songs and albums are not visible in the interface.

## System information

 * **Airsonic version**: 10.1.1
 * **Operating system**: Ubuntu Server 16.04
 * **Proxy server**: None
 * **Client**: Google Chrome (Windows, latest version) & UltraSonic app (Android)

## Additional notes

All my music is stored on a seperate Windows server and available to Airsonic via `fstab` mounts. All folders added to Airsonic will show proper album/song info in the interface, except this one.
I thought it might be related to permissions or tag-data in the files, but nothing about that has changed in the last weeks and yesterday I was able to play music from this folder without problems.

This issue occured overnight.

**Comment by muff1nman** [comment]
Can you double check your user permissions within the airsonic settings? Make sure that your user has the music folder selected.

**Comment by Denniz** [comment]
I've checked the user permissions for that folder and the user has the correct permissions to view the folder.

I managed to fix the issue by renaming my `airsonic`  folder on the server to get a clean install **again**. After finishing the initial configuration and setting up only the one music folder causing problems, the application scanned the files and I was able to see and play them in both the browser and Android app.

I'm still not sure what caused the problem in the first place and why my first attempt at resetting the database (as described initially) didn't fix this.

Anyway, everything seems to be working properly at this point. So I will close this ticket.
